src/main/kotlin/snyk/common/CustomEndpoints.kt

```diff
@@ -21,12 +21,14 @@ fun toSnykCodeApiUrl(endpointUrl: String?): String {
         uri.isDev() ->
             endpoint
                 .replace("api.", "")
+                .replace("/v1", "")
```

Comment:
These changes always make me anxious without unit tests. Could you please add some?